cmd/oras/root/file.go

```diff
@@ -62,3 +61,13 @@ func loadFiles(ctx context.Context, store *file.Store, annotations map[string]ma
 	}
 	return files, nil
 }
+
+func getPathName(path string, root string) (string, string) {
```

Comment:
I tried to come up with a better name for this function but failed. (ChatGPT failed, too) 😶‍🌫️